runelite-client/src/main/java/net/runelite/client/config/ConfigManager.java

```diff
@@ -514,6 +516,18 @@ static Object stringToObject(String str, Class<?> type)
 			int mods = Integer.parseInt(splitStr[1]);
 			return new Keybind(code, mods);
 		}
+		if (type == WorldPoint.class)
+		{
+			String[] splitStr = str.split(",");
```

Comment:
You can just use : as delimiter, like Point and Rectangle use

Reply:
Out of curiosity, is there a reason colons are preferred over commas for this?

Reply:
I think someone just made it like that and then we just followed. So no reason really, but its better to stay consistent.